packages/web-app-text-editor/src/App.vue

```diff
@@ -152,18 +152,14 @@ export default defineComponent({
             )
             if (space) {
               errorPopup(
-                // FIXME: translation
-                // $gettextInterpolate(
-                //   'There is not enough quota on "%{spaceName}" to save this file',
-                //   { spaceName: space.name }
-                // )
-                $gettext('Error when contacting the server')
+                $gettextInterpolate(
+                  'There is not enough quota on "%{spaceName}" to save this file',
```

Comment:
```suggestion
                  $gettext('There is not enough quota on "%{spaceName}" to save this file'),
```

`$gettextInterpolate` doesn't do any translations. It only inlines variables. You still need to wrap the lang string in a `$gettext` call.

Reply:
Oh boy, time for the weekend to come 😬